docs/source/models/risk_effects/bmi/index.rst

```diff
@@ -246,19 +248,20 @@ PAFs and relative risks can be pulled using the following code::
 
   pafs = get_draws(gbd_id_type=['rei_id', 'cause_id'], gbd_id=[370, 493], source='burdenator', measure_id=2, metric_id=2, year_id=2019, gbd_round_id=6, status='best', decomp_step='step5') 
 
-**Mediation with FPG**
+Mediation
+^^^^^^^^^
 
-The relative risks are then adjusted to account for mediation through FPG. 
-To find the new relative risk, this equation can be used: 
+Mediation for IHD is included for FPG, SBP and LDL-C. Data for the 
+mediation factors can be found in the csv file above. The rei_id for 
+all is 370. The cause_id for IHD is 493. The med_ids are 105 for FPG, 
+107 for SBP and 367 for LDL-C. The csv has data for individual draws 
+that will be used. 
 
-RR_adjusted = (1 - 0.149278039) * (RR_unadjusted - 1) + 1
+RR_adjusted = (1 - (MF for SBP + MF for LDL-C + MF for FPG)) * (RR_unadjusted - 1) + 1
```

Comment:
```suggestion
:math:`RR_\text{adjusted} = 1 + (RR_\text{unadjusted} - 1) * \prod_{m=1}^{n} (1 - MF_m)`

Where :math:`MF_m` is the mediation factor for each mediator :math:`m` in SBP, LDL-C, and FPG.
```

Mediation factors combine multiplicatively rather than additively! See the equation here
![image](https://user-images.githubusercontent.com/54918472/207693280-614e3904-63b8-45dc-b567-bab74d82be0a.png)

Reply:
Ohhh thanks for catching this!!